homeassistant/components/tuya/strings.json

```diff
@@ -18,61 +18,270 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "gas": {
+        "name": "[%key:component::binary_sensor::entity_component::gas::name%]"
+      },
+      "methane": {
+        "name": "Methane"
+      },
+      "voc": {
+        "name": "[%key:component::sensor::entity_component::volatile_organic_compounds::name%]"
+      },
```

Comment:
This is making translations going across integrations. I don't think we should do that.

Like using up the tree/stack to get translations that is fine (e.g., binary sensor or common), but not side-ways.